Core/KSP.cs

```diff
@@ -7,7 +7,11 @@
 using System.Text.RegularExpressions;
 using System.Transactions;
 using Autofac;
+#if NETSTANDARD2_0
```

Comment:
I am really not excited about maintaining `#if` statements. Is there no other way to do this?